extensions/dataseed/dataseed-policy/src/main/java/com/microsoft/dagx/dataseed/policy/PolicyDataSeederExtension.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ *  All rights reserved.
+ *
+ */
+
+package com.microsoft.dagx.dataseed.policy;
+
+import com.microsoft.dagx.policy.model.*;
+import com.microsoft.dagx.spi.monitor.Monitor;
+import com.microsoft.dagx.spi.policy.PolicyRegistry;
+import com.microsoft.dagx.spi.system.ServiceExtension;
+import com.microsoft.dagx.spi.system.ServiceExtensionContext;
+
+import java.util.List;
+
+import static com.microsoft.dagx.policy.model.Operator.IN;
+
+
+public class PolicyDataSeederExtension implements ServiceExtension {
+
+    public static final String USE_EU_POLICY = "use-eu";
+    public static final String USE_US_OR_EU_POLICY = "use-us-eu";
+    private Monitor monitor;
+    private ServiceExtensionContext context;
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+        monitor = context.getMonitor();
+        this.context = context;
+
+        monitor.info("Initialized Nifi Demo extension");
```

Comment:
Small issue, but I think in you mean Policy Data Seeder extension :)

Reply:
yep, you're right. will fix it :)